src/js/components/Grommet/Grommet.js

```diff
@@ -69,11 +68,12 @@ class GrommetImpl extends Component {
     const { theme } = this.state;
 
     if (userAgent) {
-      const md = new MobileDetect(userAgent);
-      if (md.phone()) {
+      if (/Mobile|iPhone|Android|BlackBerry|IEMobile/.test(userAgent)) {
```

Comment:
I think we should add a comment indicating where we got this expression from, so we can have a reference to check to update it.